Gemfile

```diff
@@ -1,3 +1,6 @@
 source 'https://rubygems.org'
 
 gemspec
+
+# introducing a local group to disallow unnecessary local dependencies installed in circle ci
+gem 'chromedriver-helper', :group => :local
```

Comment:
newline.

Also, is this gem a runtime dependency? If so, it needs to be in the gemspec. Even if it's only a development dependency, it should still be listed in the gemspec (listing gems in Gemfile and _not_ the gemspec is an anti-pattern.

I'm assuming the desire to put the gem in the Gemfile is in an effort to speed up the installation process to make CI runs faster? I'll post a snippet in the main comments on how to cache gems on circle to improve CI build times.

Reply:
So adding `chromedriver-helper`, broke circleci build, but not when running the tests locally. The base image did not need this gem I believe when running on circle.

Guess it can be moved to gemspec as is, I shall as part of future work. Cheers.

It was not for caching (do know the snippet to cache dependencies), it was simple because circle broke with `Cannot find driver error`.

Reply:
Ah, gotcha. Good luck solving that one! 😸  (I don't have any guesses other than a PATH thing?)